packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -152,7 +152,9 @@ const isValidTransaction = (tx: ParsedTransactionWithMeta): tx is SolanaValidPar
     !!(tx && tx.meta && tx.transaction && tx.blockTime);
 
 const pushTransaction = async (request: Request<MessageTypes.PushTransaction>) => {
-    const rawTx = request.payload.startsWith('0x') ? request.payload.slice(2) : request.payload;
+    const rawTx = request.payload.hex.startsWith('0x')
+        ? request.payload.hex.slice(2)
+        : request.payload.hex;
```

Comment:
Unrelated, probably just copy paste error from past
```suggestion
    const rawTx = request.payload.hex
```
I think Solana tx never starts with `0x`